CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp-main.tex

```diff
@@ -1187,7 +1188,7 @@ \subsubsection*{Current Practices}
 
 Current experiments have moved towards smaller, but still
 flexible, tiered data formats. These tiers are typically based on the
-ROOT~\cite{Brun1996} file format and constructed to
+ROOT file format and constructed to
```

Comment:
I guess the removal is because it was already cited...

Reply:
Right - 4 times?